doc: SUIT documentation on updating Nordic Bundles (#17417)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst

Instructions for these actions and further customization are described in the :ref:`ug_nrf54h20_suit_customize_dfu` page.
Additionally, you can modify SUIT components within the manifest (see the :ref:`ug_nrf54h20_suit_components` page for more information).

> In case an update of the Nordic Semiconductor firmware components is required, see the :ref:`ug_nrf54h20_suit_nordic_bundles` page.

Reviewer: ```suggestion
If an update to the *nRF54H20 SoC binaries* (provided by Nordic Semiconductor) is required, refer to the :ref:`ug_nrf54h20_suit_nordic_bundles` page.
```

> If an update to the *nRF54H20 SoC binaries* (precompiled binaries provided by Nordic Semiconductor) is required, refer to the :ref:`ug_nrf54h20_suit_nordic_bundles` page.

Reviewer: ```suggestion
If an update to the *nRF54H20 SoC binaries* (precompiled binaries provided by Nordic Semiconductor) is required, refer to the :ref:`ug_nrf54h20_suit_soc_binaries.rst` page.
```

> If an update to the *nRF54H20 SoC binaries* (precompiled binaries provided by Nordic Semiconductor) is required, refer to the :ref:`ug_nrf54h20_suit_soc_binaries.rst` page.

Reviewer: ```suggestion
If an update to the *nRF54H20 SoC binaries* (precompiled binaries provided by Nordic Semiconductor) is required, refer to the :ref:`ug_nrf54h20_suit_soc_binaries` page.
```
